GUI-pause-menu/Assets/UI/Uss/PauseMenu.uss

```diff
@@ -0,0 +1,74 @@
+.menu-button {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    font-size: 31px;
+    border-color: rgb(101, 101, 101);
+    border-left-width: 0;
+    border-right-width: 1px;
+    border-top-width: 0;
+    border-bottom-width: 0;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    color: rgba(238, 238, 238, 0.95);
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 0;
+    flex-grow: 1;
+    background-color: rgba(0, 0, 0, 0);
+    border-right-color: rgb(0, 0, 0);
+    -unity-text-align: middle-center;
+    -unity-font: url('/Assets/Fonts/Teko-Regular.ttf');
+}
+
```

Comment:
margin and padding can be simplified (margin: 0, 0, 0, 0 and padding: 12px, 0 12px, 0)

Reply:
Niet mogelijk USS, USS wordt automatisch genereert door Unity. Helaas is USS nog niet zo ver als CSS.